tools/util.py

```diff
@@ -130,16 +130,27 @@ def touch(fname):
         open(fname, 'a').close()
 
 
+def str_to_list(value):
+    return value if type(value) is list else [value]
```

Comment:
Remove - no need for ergonomics that add complexity in this internal tooling.

Reply:
ok done